sql/moz-fx-data-shared-prod/org_mozilla_broken_site_report/user_reports/view.sql

```diff
@@ -41,6 +41,10 @@ SELECT
   metrics.text2.broken_site_report_description AS comments,
   metrics.url2.broken_site_report_url AS url,
   metrics.string.broken_site_report_breakage_category AS breakage_category,
+  "Firefox Desktop" AS app_name,
```

Comment:
I've noticed there is `normalized_app_name` field in the `moz-fx-data-shared-prod.firefox_desktop_stable.broken_site_report_v1` table. It is empty for all reports though, so I've added this hard coded field. Was wondering if this is the right way to specify a product/app name?

The reporter is implemented only on desktop so far, but eventually we would like to add it to mobile as well and it would be useful to know the product name (Firefox Fenix or Focus, for example).

Reply:
Yeah, this is fine.
Usually, `normalized_app_name` is just derived from the data that Glean sends, so maybe there is something on the client-side that needs to be set.

Reply:
Thanks!

It seems that it's empty in the two tables I'm using in this view (stable and live one):
`firefox_desktop_stable.broken_site_report_v1`
`firefox_desktop_live.broken_site_report_v1`,

but it exists in `firefox_desktop.broken_site_report` table (it's `Firefox` for all reports). 

I can probably change `Firefox Desktop` to just `Firefox` then to match it.